### ISSUE TITLE
Notes: CSV file (coma vs semi-colon)

### ISSUE THREAD

**Issue by Heart1010** [OrderFlowTools/TraderLadder#16]
I just tried the new notes (csv) feature - my NT8 is set to 'English' (i come from germany) but the prices in NT8 uses a coma (,) and not a point (.) as delimter and so your csv syntax doesn't work (it works but only for full _int_ prices).

![image](https://user-images.githubusercontent.com/1481680/178368866-67c96725-c48a-48a1-bcff-148f48592761.png)

```cs
// CSV format expected is:
//INSTRUMENT,PRICE,NOTE
string[] values = line.Split(';');
```

^^^ I changed the Split character to a semi-colon so I can use e.g.  `ES;3862,25;High` in my csv file and now TraderLadder can see it.

`ES,3862,High` shows the notes
`ES,3862.25,High` shows nothing in notes column

**Comment by gemgit7** [comment]
Awesome, thanks for the fix @Heart1010! Perhaps a configurable delimiter character will make the notes reader more flexible!

**Comment by gemgit7** [comment]
Fixed in v0.3.9
https://github.com/OrderFlowTools/TraderLadder/releases/tag/v0.3.9
